massgov/openmass
# DP-21657: Add text search filter to ma-dash/feedback.

**Description:**
Add `search` field to `/admin/ma-dash/feedback` allow users to search comma separate list of strings.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-21657


**To Test:**
* visit `/admin/ma-dash/feedback`
* Add some search strings to the new `Search` text field
  * for example: `/admin/ma-dash/feedback?search=fleece&watch_content=0` should return 2 results
  * and: `/admin/ma-dash/feedback?search=fleece%2C%20underwear&watch_content=0` should return 4 results 

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
